### ISSUE TITLE
He Who Over Comes design.

### ISSUE THREAD

**Issue by Morningstar88** [Microflow/Eagle-Shirts#11]
We need 5 designs for a good crowdfunder. Don't worry, we don't need to make plates for any of these, just get some good jpgs so people can choose a design. The first actual plates will be the danshui design. 

*

## He Who Over Comes

Take the font from the Made to Reign TShirt. 

he
who
over
comes

In the red pallete, on a white TShirt. Purple monochrome will work for this too.

![image](https://www.schemecolor.com/wp-content/uploads/red-wine-color-scheme.png)

If we are really gutsy, we can do

sHe
who
over
comes

*

What is overcome in Chinese?

克服
克服

Does that sound good or cheesy?

**Comment by Morningstar88** [comment]
# Three Color Indigo Palette (Can add more indigo or a contrast color)

https://coolors.co/220066-350d88-170a1c

**Comment by Morningstar88** [comment]
## 4 Color Indigo (I like this one)

https://coolors.co/220066-350d88-410f8a-3d298e

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/85727181-61c40580-b729-11ea-83be-9b811da9c532.png)
I think this is what I have right now... The font looks like arial. It doesn't look the same without the crown or bible verses...

**Comment by Morningstar88** [comment]
That is very different from the made to reign shirt.

If you are pushed for time, we can focus on the danshui design, in a
monochrome burgundy palette, on white TShirts. I am here for a few hours if
you want to discuss that.

**Comment by taiwanluthiers** [comment]
I don't exactly know the font they used for that made to reign shirt. I can try to ask. It looks like Arial but much thicker.

Do you have a suggestion as to what to add? A picture or a bible verse?

**Comment by Morningstar88** [comment]
I think we should focus on one shirt only for printing... It takes a LOT of
time to get the design and printing right, as you know.

I just asked on Reddit:

https://imgur.com/ICrx9nT

The pic reminded me what a cool TShirt that is. Good work.

**Comment by taiwanluthiers** [comment]
Let's focus on this design. I am sure there's ways we can refine it.

However overcome is one word, not 2 word. It sounds weird as 2 words. 

I am writing a one page marketing plan along with explanations... I'm trying to do something about the customer situation. I was told my answers lack depth and I really don't know what I can do to add depth. Note this is a 2 pronged attack, on one hand I can print T shirts with their design for churches or organizations, but on the other hand it's what we're doing now, coming up with a design that people will like. 

[Who is the target market.docx](https://github.com/Microflow/Eagle-Shirts/files/4832257/Who.is.the.target.market.docx)

**Comment by Morningstar88** [comment]
Here's the Reddit link:

https://www.reddit.com/r/identifythisfont/comments/hfnug9/is_this_arial_black_or_helvetica_bold/


It might be a Helvetica or Arial variant. Black, bold or something. Note
that it's left biased.

*

With a burgundy palette, we may also want to look into lettering effects.
Gradients etc. I know we discussed it before, but your skills have improved
a lot. So let me know what monochrome gradient effects you can do.

**Comment by taiwanluthiers** [comment]
I'm thinking of a sort of a color fade. So then this uses only one or two slide (a white underbase and then a color coat).

I can definitely do this.

**Comment by Morningstar88** [comment]
I know overcomes is two words, but with one word it would cut right across
people's spleen/belly button and look awful.

Once we know the font it will be easy to do any of the designs I
brainstormed.

We can add a simple burgundy cross. Nothing else.

*

We can discuss marketing later.

**Comment by Morningstar88** [comment]
A color fade looks good in my mind. You mean from dark burgundy to light,
across a single letter?

**Comment by taiwanluthiers** [comment]
across the entire field

**Comment by Morningstar88** [comment]
It would be better across single letters and words. Fading across the whole
field could look tacky.

Imagine this:

Split the words into 4 separate plates, and do a different fade across each
word. He and Over from dark to light (left to right) Who and Comes the
opposite direction.
 Can you see that? All in burgundy.

**Comment by taiwanluthiers** [comment]
I get what you mean. keep in mind I need one plate for white underbase (you can't do this without it if you are printing onto a dark garment) and a mat-down plate to prevent fibers from sticking up. So keep it under 3 plates if possible. In fact if we are going to do a dark to light and another light to dark, we can do this with 2 plates.

**Comment by Morningstar88** [comment]
OK got it. I have moved your comment to the screenprinting effects thread.

BTW, the font is HELVETICA BLACK. Can you get hold of it? I think this
should be our default font for the next few months.

Now let's focus on awesome lettering effects, good sourcing and beautiful
words.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/85752167-c25d3d80-b73d-11ea-8e4e-dc61dd9994de.png)
I just got it

**Comment by Morningstar88** [comment]
Really impressive and exciting. Can you bias it to the left like the made
to reign shirt?

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/85753218-a0b08600-b73e-11ea-83b5-481a21e81d69.png)

**Comment by taiwanluthiers** [comment]
I have a suggestion: put "he who" as the first line. That way you can put a picture where the space is.

**Comment by Morningstar88** [comment]
better.

We will work on subtle alignments later.

Do you remember that guy in Zhongxiao Xinsheng Cafe, who had the red
lettering on a white hoodie? I would swear that is Helvetica black.

*

Would any of the Chinese translations of overcome fit on this shirt? I
think one character would look lush.

Can you pick one that doesn't sound too cheesy? I will trust you to do
that. 克服???

overcome
克服, 克, 攻下, 驯
conquer
征服, 战胜, 克服, 胜利, 侵占
surmount
超越, 克服, 战胜, 突破, 攀越
endure
忍受, 挨, 忍, 承受, 经受, 克服

**Comment by Morningstar88** [comment]
ok try it... that could work.

**Comment by taiwanluthiers** [comment]
I can't think of a way to do Chinese without it looking weird. 

![image](https://user-images.githubusercontent.com/66872000/85754317-8aef9080-b73f-11ea-8e8a-ed777e45f3b3.png)

**Comment by taiwanluthiers** [comment]
We could increase the line spacing or decrease character spacing so we can fit some icon in the middle... I'm really tempted to do Soviet Union symbol but that would not be very Christian at all...

**Comment by Morningstar88** [comment]
a sickle. ha.

Don't worry about it looking weird. Just tell me if the word 克服 fits or
not. Try and choose one. We can try with and without Chinese.

I don't think it looks good with three lines. The four line version has a
natural curve and flow.

**Comment by taiwanluthiers** [comment]
克服 sounds good.

yea I would agree with you there...

**Comment by Morningstar88** [comment]
Okey...

Next step, can you indent He by about 6%, Who by 4%, Over by 2% and leave
comes as it is.

So we need a very subtle gradient on the left. I am trying to do it in CSS
now, but you can try there.

I am busy with the website in the other window... you are better with
photoshop. I've also found some other awesome fonts.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/85757677-444f6580-b742-11ea-86a7-f495cf7c6b76.png)

Like this?

**Comment by Morningstar88** [comment]
yes, you are on the way...

Can you tell me what numbers you were using (5px or what) so we can fine
tune it. What number and unit are you using on each word?

And do you think it works like that?

**Comment by Morningstar88** [comment]
I mean, can you tell me something like?

he (7px)
who (5px)
over (3px)
comes (0px)

or are you using percentages?

**Comment by taiwanluthiers** [comment]
I can't use percentages... I just put a space in front of the "he" and set kerning to 200
I did the same with "who" but did not add the space in front
I set the kerning for "over" to 75

**Comment by Morningstar88** [comment]
If you use spaces it's hard to make micro-adjustments. Can you do a version
with kerning only?

BTW can you get hold of Frutiger font? I think Adobe use it. Not a
priority, but it looks really good. It would work for us on another shirt.

**Comment by taiwanluthiers** [comment]
Max I can do with kerning is 200. I can't go higher than that, so I had to use spaces...

**Comment by Morningstar88** [comment]
ok, can you bring the he back a little? To the left? 

It's too indented and looks wrong. Try and make a smooth diagonal line... if you get me. 

Can you do kerning 180 on he? Or is it multiples of 20?

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/85762754-a316de00-b746-11ea-94ba-fd2759bc39f3.png)

**Comment by Morningstar88** [comment]
Are you using those cm numbers up top?

does x refer to he?

If that is right, we can really fine tune it... but it's looking good now.

**Comment by taiwanluthiers** [comment]
the X and Y refer to position for the entire text box. Do not mess with it.

The number you are looking for is the one that says V|A which adjusts kearning

**Comment by Morningstar88** [comment]
ok got it

**Comment by Morningstar88** [comment]
You can use this font for kefu.. Screengrab it if you want:

https://docs.google.com/document/d/1w9_tokpgWYXwa0gYsnY6Suow4Y7rmz6aQAMXiyEsAjI/edit


I think that is the default Google docs font

**Comment by Morningstar88** [comment]
## Colors

For the colours, can you imagine if the Google logo was in 3 shades of
Burgundy?

So we would split three burgundy colors between 3 different plates/svgs. Do
you get that?

Please don't do it yet, as we have to get it perfect at jpg level before we
make the 3 svgs, otherwise it wastes time

*

So you need underbase white, and have 3 colors spare, right?

**Comment by Morningstar88** [comment]
## Multi colored He Who Overcomes Alternate Design

Now image the he who overcomes design in actual Google colors, for example.
Forget about the ripping off brands issue for now... we will choose our own
distinctive palette.

Once we get the plates made we can swap the colors after washing, right?

We'll still keep it on white.

So one design burgundy, the other multi colored.

**Comment by Morningstar88** [comment]
Some of these palettes are interesting. I will choose one later, but lets
stick with burgundy for now:

https://www.google.com/search?q=blue+orange+red+palette&sxsrf=ALeKk023TP1voX0q7c5DncH9mPRLxl-5NA:1593106054334&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjEytTDvp3qAhWMGKYKHXnaBbcQ_AUoAXoECA0QAw&biw=1920&bih=898

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/85952471-27ae6a00-b99c-11ea-9b9c-5e5f45036bd4.png)
He Who Overcomes JPG

**Comment by Morningstar88** [comment]
I agree we don't have to change the font.

Tonights focus is on getting a beautiful burgundy jpg,

**Comment by taiwanluthiers** [comment]
Would you like for me to manually change the color to burgundy?

**Comment by Morningstar88** [comment]
yes that would help.

Imagine if the google design was 3 shades of burgundy, instead of
multi-colored.

So we will call the colours B1 B2 and B3.

B1 is darkest..

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85952906-08fda280-b99f-11ea-9c21-2de6c3ff65f4.png)

How is this?

**Comment by Morningstar88** [comment]
not yet.

Each letter needs to be a clear different shade, not faded.

Hang on, I will get the palette.

*

BTW, what percentage of people in your church are Chinese vs Foreigner? Do
we need to translate the Patreon to Chinese?

**Comment by taiwanluthiers** [comment]
I think it should be in chinese, but I'd say about 25% speaks english fluently

**Comment by Morningstar88** [comment]
aha good, got it.

Chinese or English above?

**Comment by taiwanluthiers** [comment]
I'd say Chinese first

Can you show me an example? I went by the pallete you provided at the very top (since I didn't know if that was edited)

**Comment by Morningstar88** [comment]
Can you use these three hexcodes?

4b0a0e
650e15
871325

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85953165-fe440d00-b9a0-11ea-8841-ea4cf3596275.png)

Ok I tried this but it doesn't really look that different, not to mention I'd have to use 3 different screens.

**Comment by taiwanluthiers** [comment]
Also remember I will be mixing the color by hand, I won't have the benefit of mixing a color to some hex code that a computer could get exactly right every time... (paint stores have those kind of machines)

**Comment by Morningstar88** [comment]
We need to use 3 screens to get a quality shirt. The point is to do 3
shades of burgundy, to make it subtle.

Did you use those exact hexcodes? Maybe I need to separate the colors more.

I also have an idea for a multicolored version.

**Comment by taiwanluthiers** [comment]
I used the exact hexcode... that's what it gave me.

Let me know about the multi color version

**Comment by Morningstar88** [comment]
Yes, we can try burgundy later.

**Comment by taiwanluthiers** [comment]
Got it.

**Comment by Morningstar88** [comment]
What about something close to this?

https://coolors.co/c6d8ff-71a9f7-6b5ca5-72195a-4c1036

**Comment by Morningstar88** [comment]
use the three colors on the right. It's slightly monochromatic, but much
more separated.

Three colors:

https://coolors.co/6b5ca5-72195a-4c1036

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85953506-5e3bb300-b9a3-11ea-914f-ca07fb75461d.png)

Tell me how I can change the color order...

**Comment by Morningstar88** [comment]
Made the purple (left color) a bit lusher.

The other colors are the same.

We could do the Patreon with 2 or three options.

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85953520-79a6be00-b9a3-11ea-8492-7145c5455c79.png)

oops I missed a letter..

**Comment by Morningstar88** [comment]
That's good. You got what I meant.

Just please revise the purple for now.

**Comment by taiwanluthiers** [comment]
can you provide a hex code for reference?

**Comment by Morningstar88** [comment]
I like the diagonal slant with the purple. You have a very good eye for
design...

Can you make the S red?

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85953562-e3bf6300-b9a3-11ea-9b6d-aae061550b0a.png)
Like this? 

This will require 4 screens

**Comment by Morningstar88** [comment]
491D61

**Comment by Morningstar88** [comment]
sorry, my mistake.... we will rename the colors to avoid confusion.

Just give me your original version but change the purple to 491D61.

You had it right the first time

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85953641-7fe96a00-b9a4-11ea-993e-904d08ea27bd.png)

**Comment by Morningstar88** [comment]
better. Still needs more work though.

Any ideas to spice it up? Any better palette ideas?

**Comment by Morningstar88** [comment]
can you switch the colors on the O and the V of over?

**Comment by Morningstar88** [comment]
we can produce three versions and get the people in your church to decide.

**Comment by Morningstar88** [comment]
can you switch the the E and the S in comes

**Comment by Morningstar88** [comment]
we could try this too. But I have to adjust the shades later.

This is a second version

https://coolors.co/e1711b-4771bb-a1083d

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85953807-81676200-b9a5-11ea-8678-b70a4a6ad9a4.png)

I think let's call it a day... It's getting late.

**Comment by Morningstar88** [comment]
ok. That looks kind of cool. What do you think?

It will take about 5-10 hours of focused work to get a Patreon ready, but I
think we could finally monetize if we did that.

Are you comfortable with showing the Patreon to people at church?

**Comment by taiwanluthiers** [comment]
![111](https://user-images.githubusercontent.com/66872000/85953839-d73c0a00-b9a5-11ea-8e54-f5e931266c01.png)

Other version

**Comment by taiwanluthiers** [comment]
I will show it once the final version is done...

**Comment by Morningstar88** [comment]
Hmm, that palette is a bit too tacky. I will tone it down... but I like the
general multicolors.

In the burgundy one, can you add 克服 next to the WHO, but down a bit. Just
use your feeling.

Tomorrow, just open this thread on github, and slowly go through my
messages one by one...

**Comment by Morningstar88** [comment]
We only get one shot at church, so lets make it good.

Goodnight.

**Comment by taiwanluthiers** [comment]
I will do that tomorrow morning... 

I agree that alternate color is too much of a rip off of google. It's really obvious, but it looks good.

**Comment by Morningstar88** [comment]
In general, cos we have white as a background, we can afford to use darker
shades, which look less cheesy.

I was thinking of swapping the orange for dark forest green.

Any other ideas you have are welcome.

**Comment by Morningstar88** [comment]
## Third Palette

https://coolors.co/e1711b-4771bb-a1083d

**Comment by Morningstar88** [comment]
We can ask for help on Reddit. Just writing myself a note here.

**Comment by Morningstar88** [comment]
We can split the colors across individual letters to get less Googlish.
i.e.. have the v as two or three colors.

It will take more work, but might be good.

**Comment by Morningstar88** [comment]
we can think about splitting the letters vertically in half, to suggest a
mirror, or duality.

1) If you have time, try and split every letter down the middle.

2) Another option is to only split the symmetrical letters down the middle.

I know this is a lot of mails, but this is the design process. You have to
beat your head against the wall until you get something excellent.

If you get lost on gmail, find the original thread on github.

**Comment by taiwanluthiers** [comment]
Splitting the letters will take a lot more time to do compared to each individual letters. I would have no way to do this quickly in paint. Also in illustrator I would have to find a way to do this as well...

**Comment by Morningstar88** [comment]
Yes, it's more time... which means a better design, which means more sales,
and more money, raised on Patreon.

We need a five star design that will blow people's minds for you to
showcase.

Expect this design process to take 5-10 hours. That is life.

It's better that you use illustrator. I am getting the website ready in
another window.

We also need to work on this together. I need your feedback and energy.

**Comment by taiwanluthiers** [comment]
I'll work on this tomorrow...

**Comment by Morningstar88** [comment]
No rush, just plod methodically through my emails. I maye write them up

You can paste my mails here and we'll make a doc if that's easier for you.

https://pad.riseup.net/p/2wIVxP0hCVpXrZg4-rrR-keep

Please confirm if you want to do it that way.

**Comment by Morningstar88** [comment]
If you want to know how difficult it is for me, look at the website Im
building, and scroll down in the HTML window slowly. There is 1300 lines of
code just for that tiny site.

https://codepen.io/Teeke/pen/rNxNZjq

**Comment by taiwanluthiers** [comment]
I would prefer that... a simple to do list, that way I don't miss stuff...

**Comment by Morningstar88** [comment]
OK, I get that.

But part of the design process is going back and forth and sharing ideas. I
need your feedback. I can't make todo lists on *my own.

It's also your responsibility to stay on top of threads, using the github
interface, not just mails.

https://github.com/Microflow/Eagle-Shirts/issues/11

So we will brainstorm on Github and then make docs.

**Comment by Morningstar88** [comment]
I mean, on *my own

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/85970742-f36b9580-b9fd-11ea-992d-300280160ac7.png)

Just working color... please come up with a palette when necessary

**Comment by Morningstar88** [comment]
That is starting to look good. I'll darken the palette a little...

Can we add 克服 somewhere? It needs to be parallel to the "who" and up a little. Just use your judgement. 

Please use this actual thread instead of mails. 

https://github.com/Microflow/Eagle-Shirts/issues/11

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86022928-94d60400-ba5d-11ea-8b9f-44b4422cd047.png)

How is this?

**Comment by Morningstar88** [comment]
Better.

- Can you color it red.
- Plz move it a little to the right so it's over the right side of the e,
- Keep the horizontal level the same for now.

I drew a line where the bottom left of the character should be.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86024234-517c9500-ba5f-11ea-9aa3-dd56f106cf4b.png)

**Comment by Morningstar88** [comment]
Getting better. 

any more ideas?

Do you think we should split the h or the m?

**Comment by Morningstar88** [comment]
Maybe the h and the m could have slightly diagonal splits. If you think that works we could try it. I can draw it

**Comment by taiwanluthiers** [comment]
I didn't split the non-symmetrical characters because they wouldn't look right split...

**Comment by Morningstar88** [comment]
We can split them in different places, and diagonally. You're right a symmetrical split wouldn't look good. Let me try. 

We should leave the c, e and s alone. 

*

Can you go up the thread and reply to the codepen I left.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86025813-40cd1e80-ba61-11ea-8514-9a4130ab91a6.png)

**Comment by Microflow** [comment]
Good try but we need to follow the curve of the letters more.

I drew the split on the h and the m.

Try and follow it closely if you can.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86026543-26477500-ba62-11ea-847d-3202e2eec2ea.png)

indigo color scheme

**Comment by taiwanluthiers** [comment]
I can't see your drawing. You can drag and drop it directly to the post.

**Comment by Microflow** [comment]
Oh right, sorry. I wasn't sure how to do it. I emailed it to you.

![imageedit_2_2525351125](https://user-images.githubusercontent.com/15410443/86026686-5262f600-ba62-11ea-871c-4c3e5c67db2a.gif)

**Comment by Microflow** [comment]
Here's the one with the m and h. 

![imageedit_2_8805303980](https://user-images.githubusercontent.com/15410443/86026862-8ccc9300-ba62-11ea-9036-5f0cd577264f.gif)

**Comment by Morningstar88** [comment]
![imageedit_3_2633935364](https://user-images.githubusercontent.com/15410443/86026961-abcb2500-ba62-11ea-8813-b126d38a7a8f.gif)

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86027047-ca312080-ba62-11ea-9fd1-3bef475d8e9f.png)
Indigo color, h split

**Comment by Morningstar88** [comment]
So plz leave the h of h alone. just focus on who h and comes m

The indigo scheme looks really cool, but the colours need to be separated more. I'll look for a new palette. 

It's hard to judge with those murky colors, can you go back the the other palette for now? I will get an indigo palette together.

The split angle on the h is right, but it needs to come down lower.

**Comment by Morningstar88** [comment]
Try and follow my line on the h exactly.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86027459-52172a80-ba63-11ea-9cd1-1058b8b79665.png)

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86027613-825ec900-ba63-11ea-8dfe-12a9756ac1e7.png)

**Comment by Morningstar88** [comment]
The m is good.

ah. Got it. the h looks good now too.

**Comment by Morningstar88** [comment]
Any ideas?

We have to find a good pallete now. That one is too tacky.

**Comment by taiwanluthiers** [comment]
I'm not sure. But maybe just something that's complimentary but not too close in shade (because now I can't tell the difference)

**Comment by taiwanluthiers** [comment]
complimentary means on the same side of the color wheel, like yellow to green to blue, red to orange to yellow, red to purple to blue, etc.

**Comment by Morningstar88** [comment]
We need a broader indigo palette.

I think we need 3-4 possible designs for the Patreon. Then we can ask
people what they want.

**Comment by Morningstar88** [comment]
I need to understand the colour wheel more.

**Comment by taiwanluthiers** [comment]
Indigo is good but better would be indigo, light purple, then bright red

**Comment by taiwanluthiers** [comment]
or actually, magenta, indigo, then cyan

**Comment by taiwanluthiers** [comment]
I am going to head to bed... getting tired. Let's think about color some other time.

**Comment by Morningstar88** [comment]
Im thinking along those lines.

Try this for green and blue.. Then lets look at your idea.

We might end up with one green and blue design, one purple monochrome
palette, and one burgundy monochrome palette.

002500
006648
83CCEA

**Comment by Morningstar88** [comment]
Sure.

That was good work today. I will look at palettes.

**Comment by Morningstar88** [comment]
Tomorrow please catch up on lost messages in the thread before we
talk about anything else. There are lots of missed messages.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86029485-cf439f00-ba65-11ea-8a60-8de04f559f64.png)

**Comment by Morningstar88** [comment]
That doesn't look too good, but it's my fault for choosing that palette.

Can you swap the light green with the light blue?

Do it tomorrow if you're tired. But I realized this palette is nothing
special.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86076473-b2868600-babc-11ea-9f6f-dd547fb577d0.png)

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86076519-c92cdd00-babc-11ea-8323-51e8daebd2db.png)

**Comment by Morningstar88** [comment]
Interesting. It's on the way. I think we need a darker blue.

The separation between the light and dark green is clear.

Good work.

**Comment by taiwanluthiers** [comment]
let me know when you have a better scheme. I think the blue kinda clash with the rest.

I'm going to sleep a little early today so if I don't respond I am probably asleep...

**Comment by Morningstar88** [comment]
We can work something out. I think we're close to a good design.

I will be here for a few hours if you want to work on the design, and the
Patreon pitch.

We can also work out a new way of working so we don't miss messages.

**Comment by Morningstar88** [comment]
https://coolors.co/641220-6e1423-85182a-a11d33-a71e34-b21e35-bd1f36-c71f37-da1e37-e01e37

**Comment by Microflow** [comment]
[image: purple.png]

**Comment by Microflow** [comment]
[image: red.png]

**Comment by Morningstar88** [comment]
![red](https://user-images.githubusercontent.com/15410443/86257683-79e5ca00-bbec-11ea-8585-e8dd19dcf1ed.png)

**Comment by Morningstar88** [comment]
![purple](https://user-images.githubusercontent.com/15410443/86257692-7eaa7e00-bbec-11ea-8f7b-8a4842179cf7.png)

**Comment by Morningstar88** [comment]
# 1 red 2 purple

![1 red 2 purple](https://user-images.githubusercontent.com/15410443/86259196-4dcb4880-bbee-11ea-9ad5-37aba6010026.png)

**Comment by Morningstar88** [comment]
# 1 Red Two Lighter Purple

![purple (1)](https://user-images.githubusercontent.com/15410443/86260153-7a339480-bbef-11ea-95f7-baed4ede94db.png)

**Comment by Morningstar88** [comment]
## Lighter Middle. Other Colors the same

![change middle](https://user-images.githubusercontent.com/15410443/86261740-8c163700-bbf1-11ea-9e70-58e3ff102411.png)

**Comment by Morningstar88** [comment]
# Ruby, Electric Purple, Indigo.

Changed Burgundy to Ruby on Tylers advice

![ruby](https://user-images.githubusercontent.com/15410443/86262176-20809980-bbf2-11ea-91bd-8111f7645c0d.png)

**Comment by Morningstar88** [comment]
# Electric Purple in the Center

![electric purple](https://user-images.githubusercontent.com/15410443/86262572-a3095900-bbf2-11ea-91df-e196050cb0e4.png)

**Comment by taiwanluthiers** [comment]
# Han Purple in the Center.

![aaa](https://user-images.githubusercontent.com/66872000/86263005-2b87f980-bbf3-11ea-9aa3-3706bcd59396.png)

https://coolors.co/a30023-650fe6-47007a We have a winner

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86264042-97b72d00-bbf4-11ea-9ed9-fb08056664c5.png)

**Comment by Morningstar88** [comment]
hmmm. too much red

**Comment by Morningstar88** [comment]
Can you make the R dark purple?

and the S dark or light purple?

**Comment by Morningstar88** [comment]
And the chinese text could go from dark purple to light... if you think
that works.

**Comment by taiwanluthiers** [comment]
no what happened is the hex codes didn't translate properly into illustrator. I am fixing that now

**Comment by Morningstar88** [comment]
that's good. The design still needs a bit of work

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86265030-ddc0c080-bbf5-11ea-9c96-af4f171adc75.png)

**Comment by Morningstar88** [comment]
are they really the colors? They look tacky

**Comment by Morningstar88** [comment]
I wonder if all purple (three shades) might work)

The red is sticking out like a sore thumb,

We could try two designs: mono red and mono purple.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86265381-4c9e1980-bbf6-11ea-86f1-38981a353b35.png)

**Comment by taiwanluthiers** [comment]
the red kinda clashes

**Comment by Morningstar88** [comment]
Another option is to have the English text 2 shades of purple.

And the Chinese text red.

**Comment by taiwanluthiers** [comment]
that will work better

**Comment by Morningstar88** [comment]
yes. so we can change it to purple,

or move it to the chinese text.

**Comment by Morningstar88** [comment]
I swear the lighter purple is different

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86265605-9129b500-bbf6-11ea-82de-be4af7cf1191.png)

**Comment by Morningstar88** [comment]
better, starting to get exciting

**Comment by Morningstar88** [comment]
I think we need to split more letters.

The R, for example. And even the S, horizontally.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86266782-4c068280-bbf8-11ea-809c-21b8762fad14.png)

**Comment by Morningstar88** [comment]
Better.. But the light purple is still tacky. And we need to split the
letters more.

I also think the letters on the shirt shouldn't be too big,. Maybe an inch
high

**Comment by Morningstar88** [comment]
# Found this online

![28f36d2c2da2891dd455193f85cbcec2](https://user-images.githubusercontent.com/15410443/86268326-9a1c8580-bbfa-11ea-9bed-eb41844e047d.jpg)

**Comment by Morningstar88** [comment]
# Mosaic Lettering

https://www.google.com/search?q=cubism+mosaic+lettering&tbm=isch&ved=2ahUKEwjPnbSMvKzqAhXTIaYKHTLDDI0Q2-cCegQIABAA&oq=cubism+mosaic+lettering&gs_lcp=CgNpbWcQAzoECCMQJ1ClR1jaUmDpU2gAcAB4AIABJogB7AGSAQE3mAEAoAEBqgELZ3dzLXdpei1pbWc&sclient=img&ei=Pbn8Xo-LKNPDmAWyhrPoCA&bih=898&biw=1903&hl=en

**Comment by taiwanluthiers** [comment]
it looks good but I am not sure how to implement it...

**Comment by Morningstar88** [comment]
## New Design. July 2nd.

Last night I prayed a little for inspiration.

- Take out the "he who" and focus on the overcomes, in Helvetica Black. 
- First design, just have the palette on the chest of the TShirt, as a solid block of colour. 
- Overcome below, in deep purple Helvetica Black. 
- Dark purple cross on the back if funds permit double sided. 
- Maybe use the Chinese overcome writing but we need a better font. 


I will make a draft now. 

# Han Purple in the Center.

![aaa](https://user-images.githubusercontent.com/66872000/86263005-2b87f980-bbf3-11ea-9aa3-3706bcd59396.png)

https://coolors.co/a30023-650fe6-47007a We have a winner

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86318667-219ede80-bc65-11ea-8fa7-1543799fd8b6.png)

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86318992-f4066500-bc65-11ea-955b-4d2bfe32ac00.png)

**Comment by Morningstar88** [comment]
Getting there. 

The Overcome should be with a large purple cross.

Maybe the Kefu text should be light purple. Lets keep this ultra simple.

**Comment by Morningstar88** [comment]
## Mark Rothko

Last night I prayed a little for inspiration. I got images of solid blocks on color in my mind. Check out Mark Rothko. He makes these huge minimal paintings. They are like 3-4 meters high. I want to do something a little similar...

https://www.google.com/search?q=mark+rothko&sxsrf=ALeKk03tsPwGG1grXGw9puqgXiGUrMhPdg:1593666791180&source=lnms&tbm=isch&sa=X&ved=2ahUKEwi03Om3563qAhUNqpQKHYZJABIQ_AUoAXoECB0QAw&biw=1920&bih=947

**Comment by Morningstar88** [comment]
# Mark Rothko Light Earth And Blue 1954 oil painting reproduction on canvas

www.pauloeuvreart.com

![mark-rothko-light-earth-and-blue-1954](https://user-images.githubusercontent.com/15410443/86319192-873f9a80-bc66-11ea-8acf-7bb6d85a2c4b.jpg)

**Comment by Morningstar88** [comment]
# Mark Rothko Interpretation Orange Blue Art Print by jamespeart

![mark-rothko-interpretation-orange-blue-prints](https://user-images.githubusercontent.com/15410443/86319251-b2c28500-bc66-11ea-9ffe-9656d5d55dd2.jpg)

**Comment by Morningstar88** [comment]
# Bridget Riley

## Royal Scottish Academy, Edinburgh, review: eye-scrambling insights into the workings of a truly great artist

Check out what Bridget does with color here.

https://www.telegraph.co.uk/art/what-to-see/bridget-riley-review-royal-scottish-academy-edinburgh-eye-scrambling/

**Comment by Morningstar88** [comment]
@taiwanluthiers 

could the overcome text be all lowercase: overcome

**Comment by Morningstar88** [comment]
# Mark Rothko. Red on Red

![Mark Rothko Red on Red](https://user-images.githubusercontent.com/15410443/86319511-42683380-bc67-11ea-80cc-ea8769849916.png)

**Comment by Morningstar88** [comment]
## Mark Rothko video

So I want to do something with simple blocks of color. Not sure how yet. 

It's hard to see how powerful his paintings were until you see them live. They are huge. Check out this video, to see the sense of space he got. I think, once we find a good palette, we could do something similar on a T-Shirt. 

This vid is 5 minutes long, and well worth watching. Something like that on a T-Shirt would look bold. 

https://www.youtube.com/watch?v=lPgj7lHnU10

**Comment by taiwanluthiers** [comment]
Back
![image](https://user-images.githubusercontent.com/66872000/86319773-dafeb380-bc67-11ea-8435-bd56b28c25f9.png)

**Comment by Morningstar88** [comment]
better.

1) We need to fix that light purple. It does not look good... too bright.

2) Later, we can do something interesting with the overcome text. Slice it
up or something.

3) The top of the cross is too short. Can you extend it?

**Comment by Morningstar88** [comment]
# Mark Rothko T-Shirts 

@taiwanluthiers. What do you think of these? We will do our own style, and incorporate Christian imagery... but I think they look really cool. Better than anything in Taipei. Do you know how to get those faded edges? I will research how he did it. 

https://www.google.com/search?q=mark+rothko+tshirt&tbm=isch&ved=2ahUKEwiDg8y4563qAhUE15QKHbEcCH4Q2-cCegQIABAA&oq=mark+rothko+tshirt&gs_lcp=CgNpbWcQAzIGCAAQChAYOgIIADoECAAQQ1DL-kNY0IFEYMCCRGgAcAB4AIABJ4gB5AGSAQE3mAEAoAEBqgELZ3dzLXdpei1pbWc&sclient=img&ei=6Gz9XoPvL4Su0wSxuaDwBw&bih=947&biw=1920

**Comment by taiwanluthiers** [comment]
I'm not sure... I have to find out how to do this in illustrator.

**Comment by Morningstar88** [comment]
Find out how to do what? The Mark Rothko paintings, slicing up the text?

I mean we could maybe do something like split the colors, but more
complicated...

**Comment by Morningstar88** [comment]
Oh, I guess you meant the faded edges. Im researching it now..

**Comment by Morningstar88** [comment]
## How to fade edges in Adobe Illustrator

https://www.google.com/search?sxsrf=ALeKk014g_05B0bmXpBNFcK9zIS2oOcVmw%3A1593668594309&ei=8nP9Xuy3EoyZr7wP0_Sv0Ag&q=how+to+fade+edges+adobe+illustrator&oq=how+to+fade+edges+adobe+illustrator&gs_lcp=CgZwc3ktYWIQAzIECCMQJzoECAAQR1DGCljGCmDtC2gAcAF4AIABP4gBP5IBATGYAQCgAQGqAQdnd3Mtd2l6&sclient=psy-ab&ved=0ahUKEwjs-M-T7q3qAhWMzIsBHVP6C4oQ4dUDCAw&uact=5

# Feather the edges of an object

- Select the object or group (or target a layer in the Layers panel).
- Choose Effect > Stylize > Feather.
- Set the distance over which the object fades from opaque to transparent, and click OK. Original object selected (left) and using feather effect (right)

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86321349-688fd280-bc6b-11ea-806a-845d4e033cd3.png)

**Comment by Morningstar88** [comment]
That's kind of interesting. There's so many directions we could go in. So if we take that to a print shop they will be able to replicate that fairly exactly?

The chinese text needs to be dark purple IMO. Lets play safer. Red might clash.

Any ideas what direction to go in?

**Comment by taiwanluthiers** [comment]
I would think so, but I would want to know how they do it. All I can think of is halftone conversion, with dots that are about 0.3mm in size (which is doable with silkscreen)

**Comment by Morningstar88** [comment]
The text should be smaller, and the picture bigger, IMO. Just a little. The colors need to dominate. 

Try dark purple for the first color, and lighter for the second. Our light purple is too light, so try a darker version. First character, dark purple. Second, a slightly lighter purple. 

You could round the corners of the TOP LEFT and BOTTOM RIGHT only. This would make it look like it's spinning. We could do that on the back cross too... make it look like it's a spinning shuriken.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86322095-e7d1d600-bc6c-11ea-9ea8-97a28c7af308.png)

**Comment by Morningstar88** [comment]
## Mark Rothko T-Shirts. 

These really are amazing IMO

https://fineartamerica.com/shop/tshirts/mark+rothko

I realized that copyright might have expired on his paintings, so we might be able to use them legally and ethically. I will investigate the legal issues. OTOH, we could just do our own versions. 

https://fineartamerica.com/shop/tshirts/mark+rothko

**Comment by Morningstar88** [comment]
Re: your new design. I liked the first one more. The second needs to be blurred more. Try and blur it so the white is barely visible.

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86322344-62025a80-bc6d-11ea-984a-5a7dc38bda50.png)

**Comment by taiwanluthiers** [comment]
![image](https://user-images.githubusercontent.com/66872000/86322448-9413bc80-bc6d-11ea-9565-0b9656d65a4f.png)
Feathered it a little bit

**Comment by Morningstar88** [comment]
I liked the first one much more. And as we are prototyping in a pro shop,
screen printing is not a priority these first runs. It's really the
quickest way to get to the market.

I asked on Reddit.

## Reddit Mark Rothko Question

https://www.reddit.com/r/screenprint/comments/hjrkgu/how_could_we_do_mark_rothko_style_prints_on_a/

**Comment by Morningstar88** [comment]
Still not as good as the first blurred one.

You could put a white cross in the center of those blurred pics. Make it
smallish.... It might look better a little lower.

**Comment by taiwanluthiers** [comment]
I showed the new design to my church pastor, here is what he says

As far as feedback goes..

I like the colors, but not sure what the purpose of them is. Like do they have a specific meaning for you? 
I think the fonts are kind of boring. I’d like to see them feel more like you are strong and overcoming. In particular, the chinese font feels a little thin and weak.

**Comment by Morningstar88** [comment]
That's good advice. We can take it on board...

Purple is royalty, and Red is blood. That cuts to the core of the Christian
mythos.

Don't bug him with my reply. We need a light touch. Best to contact him
once a week.
